tgui/packages/tgui-panel/styles/goon/chat-dark.scss

```
@@ -337,7 +337,14 @@ em {
   font-weight: bold;
 }
 
-.say, .emote, .infoplain, .oocplain, .warningplain {
+.say, .infoplain, .oocplain, .warningplain {
+}
```

Stylelint (17.13.0)

[error] Empty block (block-no-empty)

(block-no-empty)